src/__Private/Inspector/InspectorCLI.hack

```diff
@@ -15,6 +15,20 @@ use type Facebook\CLILib\CLIWithRequiredArguments;
 use type HH\Lib\_Private\PHPWarningSuppressor;
 use namespace Facebook\CLILib\CLIOptions;
 
+use namespace /* HHAST_IGNORE_ERROR[UseStatementWithAs] */ Facebook\XHP\Core as x
```

Comment:
@lexidor previously suggested a whitelist for UseStatementWithAs - this should be configurable, but we potentially want to whitelist Facebook\XHP by default